glide.lock

```diff
@@ -169,4 +184,63 @@ imports:
   - pkg/util/yaml
   - pkg/watch
   - pkg/watch/versioned
+- name: k8s.io/kubernetes
```

Comment:
@kadel the vendor and glide commit is different with a relevant commit message, and this is commit is after and is relevant to secrets.
We can add similar validations for other objects in subsequent PRs, but since this is relevant to secrets and testing the same, should it not be fine if this remains in this PR in this commit?